README.md

```diff
@@ -1,8 +1,13 @@
 Main Branch Status: ![CI/CD Workflow](https://github.com/Health-Education-England/tis-trainee-forms/workflows/CI/CD%20Workflow/badge.svg?branch=main)  
 Deployment Status: ![CI/CD Workflow](https://github.com/Health-Education-England/tis-trainee-forms/workflows/CI/CD%20Workflow/badge.svg?branch=main&event=deployment_status)
-# TIS Trainee Forms
+# TIS Trainee Forms Service
 
 ## About
+
+This service is used to retrieve and parse forms from an S3 Bucket, and to attach to saved forms
+the 'submissionDate' and 'lastModified' values before saving to the bucket. 
```

Comment:
I think its worth mentioning that forms are saved to S3 (not always, e.g. if they are draft) and a mongo database.